common/sgx/verifier.c

```diff
@@ -443,96 +444,119 @@ static oe_result_t _fill_with_known_claims(
             valid_until,
             sizeof(*valid_until)));
 
-        // SGX quote PCESVN
-        OE_CHECK(oe_sgx_add_claim(
-            &claims[claims_index++],
-            OE_CLAIM_SGX_PCE_SVN,
-            sizeof(OE_CLAIM_SGX_PCE_SVN),
-            &sgx_quote->pce_svn,
-            sizeof(uint16_t)));
-
         // quote QEID
+        /*
+         * IETF RAT draft requires that the UEID field be 33 bytes long, with
+         * the first byte reserved for type. The current SGX qe_id is only 16
```

Comment:
This sentence is incorrect.  It says:
>   | 0x01 | RAND | This is a 128, 192 or 256-bit random number         |
...
>    ueid-type = bstr .size (7..33)

so a 1 byte type followed by a 16 byte (128 bit) qe_id is perfectly fine without needing any padding.

Reply:
@dthaler I think the 0-padding might be needed because the draft also says
`UEIDs are variable length.  All implementations MUST be able to
   receive UEIDs that are 33 bytes long (1 type byte and 256 bits).  No
   UEID longer than 33 bytes SHOULD be sent.` ?